package.json

```diff
@@ -583,6 +628,11 @@
         "title": "%commands.SAS.authorize%",
         "category": "SAS"
       },
+      {
```

Comment:
I believe it's similar to other context menu commands that is not suitable for Command Palette. Would you add in below "commandPalette" entry to have a `"when": false` entry?

Reply:
Done in https://github.com/sassoftware/vscode-sas-extension/pull/1557/commits/149b547f1a96e2954ba0773579d4810785521a6e